### ISSUE TITLE
No se aplicó la metodología

### ISSUE THREAD

**Issue by evolentini** [monzonlucasfabricio/Testing_TP3#5]
La metodología propuesta dice que no se puede escribir ni una linea de código si no es para pasar una prueba. Este bloque de codigo no tiene ninguna utilidad, no se utiliza para nada saber si es un numero de led limite o cualquier otro, solo es importante saber si es error o no. Este bloque de código no debería de existir.

https://github.com/monzonlucasfabricio/Testing_TP3/blob/0eafca1e9f9e689f5249da376b3f8334b0c49d1c/src/leds.c#L38-L45

**Comment by monzonlucasfabricio** [comment]
Hola Esteban, yo tomé los requisitos que habíamos anotado en la clase del
día que arrancamos el TP3, por eso tengo cosas como esas que son distintas
a mis compañeros.

**Comment by evolentini** [comment]
Lucas, los requerimientos del software los di yo en las diapositivas y no había ningún relacionado con determinar si un led era un valor limite. Si había pruebas para revisar los valores de borde, pero lo que se buscaba en esas pruebas es que si prendes el led 1 o el led 16 funcione correctamente y que si prendes el led 0 o el 17 te indique un error.

**Comment by monzonlucasfabricio** [comment]
Claro, pasa que lo empecé a hacer con el vídeo de la clase y mis compañeros
dijeron algunos requerimientos que quedaron en el tintero nada más y
dijiste que era opcional hacerlos o no.
